src/client/styles/main.scss

```diff
@@ -48,3 +48,28 @@ body {
 li {
   list-style: none;
 }
+
+.fade{
+  opacity: 0;
+}
+
+.fade-appear-active {
+
+}
+.fade-enter {
+  opacity: 0.01;
+}
+
+.fade-enter.fade-enter-active {
```

Comment:
Why not nest these rules?

Reply:
removed for now, I was experimenting via react-css-transition-group but don't want to go too far with it because we haven not set the expectation on if/how this should animate